dist/algoliasearch.helper.js

```diff
@@ -10623,8 +10623,8 @@ var lib = {
    * @param {string} value the value of the refinement
    * @return {RefinementList} a new and updated list
    */
-  toggleRefinement: function toggleRefinement(refinementList, attribute, value) {
```

Comment:
An example of deprecated method. https://github.com/algolia/algoliasearch-helper-js/blob/develop/src/algoliasearch.helper.js#L307-L312 We keep them in order not to break the current implements and we'll drop them in the next version.